packages/twenty-server/src/engine/workspace-manager/workspace-migration-builder/factories/workspace-migration-field.factory.ts

```diff
@@ -87,29 +88,57 @@ export class WorkspaceMigrationFieldFactory {
   ): Promise<Partial<WorkspaceMigrationEntity>[]> {
     const workspaceMigrations: Partial<WorkspaceMigrationEntity>[] = [];
 
-    for (const fieldMetadata of fieldMetadataCollection) {
-      if (fieldMetadata.type === FieldMetadataType.RELATION) {
-        continue;
-      }
+    const fieldMetadataCollectionGroupByObjectMetadataId =
```

Comment:
grouping field migrations by object so we can have a unique migration per object with all fields keeping order of insertion. This is helping because searchVector column needs to be created after the ones that are used in the expression so we need to have a reliable way to keep the correct order.